### ISSUE TITLE
Fix pyflakes from twisted/internet

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7006]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#7006|
|Type|enhancement|
|Created|2014-02-27 15:54:16Z|
|Branch|https://github.com/twisted/twisted/tree/pyflakes-internet-7006-3|

Attachments:

* [7006-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/709/709ab2e645ba2fa720d49cd47015daa8ac786a55/ea384bb3da4b6e254264709d0d78382b62145fe2.diff) (14720 bytes) - added by adiroiban on 2014-02-27 16:15:43Z - 
* [7006-2.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/709/709ab2e645ba2fa720d49cd47015daa8ac786a55/acbe746837bf591d6603751ce563e39e68a1224c.diff) (17000 bytes) - added by adiroiban on 2014-04-07 08:55:17Z - 
* [7006-3.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/709/709ab2e645ba2fa720d49cd47015daa8ac786a55/d77af3637810942227f68bc0416135f5e1beed96.diff) (17005 bytes) - added by adiroiban on 2014-04-16 09:00:59Z - 
* [7006-4.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/709/709ab2e645ba2fa720d49cd47015daa8ac786a55/cca63bae4094eb7f1283491e811177eaf33fdbaa.diff) (19318 bytes) - added by adiroiban on 2014-04-21 08:38:59Z - 
* [7006-5.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/709/709ab2e645ba2fa720d49cd47015daa8ac786a55/c807cec337f3fa81d56d5434b3030ab4efa2ed2e.diff) (18430 bytes) - added by adiroiban on 2014-09-30 14:54:45Z - 
* [7006-6.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/709/709ab2e645ba2fa720d49cd47015daa8ac786a55/95cd0561a0b6bfd4d22cf339280675be277ddde3.diff) (19018 bytes) - added by adiroiban on 2014-11-10 13:59:00Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7006 7006
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_pyflakes_internet_7006_3 branches/pyflakes-internet-7006-3
branch_author__exarkun__adiroiban__glyph exarkun, adiroiban, glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1393516456000000 1393516456000000
changetime__1424767197887185 1424767197887185
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

All pyflakes error were fixed.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @adiroiban|
|-|-|

Thanks.

  1. The indentation change in `_sslverify.py` seems unrelated and incorrect.
  1. To fix the error about `install` in `gtk2reactor.py` I think it would be better to rename `install` to something else (perhaps `posixInstall`) and then extend the conditional with an `else` to set `install` to `posixInstall` when appropriate (replacing the `!=` with a `==` while doing this would also seem to make sense).
  1. Ditto `gtkreactor.py`
  1. A better fix for `brokenLinuxPipeBehavior` in `process.py` would be to make `detectLinuxBrokenPipeBehavior` *return* the value it computes instead of jamming it into a global.  Then the caller can jam it wherever the caller wants (like into that global).
  1. The definition of `SerialPort` in `serialport.py` looks... useless.  I guess the point of this is "if you're not on posix and you're not on win32 then we don't know how to implement serial port support for you, so we'll quietly give you this broken thing".  That seems lame.  Also, there are no platforms that are neither posix nor win32 so this lameness also seems pointless!  Perhaps this should merely be an `if-else` with no third condition and the empty `SerialPort` class can just be deleted.
  1. in `twisted/internet/tcp.py`, why not just delete the `ECONNRESET`, `ENOTCONN`, and `EINTR` imports?
  1. pyflakes (0.8.1) seems to have given up the idea of reporting locals that shadow unused globals so maybe `install` doesn't need to be renamed in `test_default.py`.
  1. I think the `epollreactor` work-around in `test_default.py` should now be replaced with a `namedModule` or whatever the testing helper that was introduced for this case is?
  1. The fix for `test_sslDHparameters` in `test_endpoints.py` is incorrect.
  1. The fix for the `inotify` import in `test_inotify` should use that testing helper too.
  1. And for the `win32process` and `qtreactor` imports in `test_qtreactor.py`
  1. And for the `sendmsg` import in `test_unix.py`
  1. I'm keen to think of another fix for the `_SendmsgMixin` in `unix.py` but I'm running out of time for this review.
  1. I think the `test_tls.py` change will conflict with trunk because `_oldtls.py` has been deleted.

Thanks again.  Sorry for the long round-trip time.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban removed owner|
|-|-|

Thanks.

> The indentation change in _sslverify.py seems unrelated and incorrect.

My bad for fixing the docstring and not updating the indentation.

> To fix the error about install in gtk2reactor.py I think it would be better to rename install to something else (perhaps posixInstall) and then extend the conditional with an else to set install to posixInstall when appropriate (replacing the != with a == while doing this would also seem to make sense).
> Ditto gtkreactor.py

Done.

> A better fix for brokenLinuxPipeBehavior in process.py would be to make detectLinuxBrokenPipeBehavior *return* the value it computes instead of jamming it into a global. Then the caller can jam it wherever the caller wants (like into that global).

Done.

> The definition of SerialPort in serialport.py looks... useless. I guess the point of this is "if you're not on posix and you're not on win32 then we don't know how to implement serial port support for you, so we'll quietly give you this broken thing". That seems lame. Also, there are no platforms that are neither posix nor win32 so this lameness also seems pointless! Perhaps this should merely be an if-else with no third condition and the empty SerialPort class can just be deleted.

Done. I have raise a NotImpleemntedError in else to let future users know
that their OS is not supported.

> in twisted/internet/tcp.py, why not just delete the ECONNRESET, ENOTCONN, and EINTR imports?

I was thinking that they are imported there on purpose so that external code can import those errors from tcp in a portable way.

I checked twisted code and no code is using those constants from tcp.py.
I have removed them.

> pyflakes (0.8.1) seems to have given up the idea of reporting locals that shadow unused globals so maybe install doesn't need to be renamed in test_default.py.

Done.

> I think the epollreactor work-around in test_default.py should now be replaced with a namedModule or whatever the testing helper that was introduced for this case is?

Done.

> The fix for test_sslDHparameters in test_endpoints.py is incorrect.

True. Done.

> The fix for the inotify import in test_inotify should use that testing helper too.
> And for the win32process and qtreactor imports in test_qtreactor.py
> And for the sendmsg import in test_unix.py

Done.

> I'm keen to think of another fix for the _SendmsgMixin in unix.py but I'm running out of time for this review.

I tried 'something new'. Please let me know if this is better.

> I think the test_tls.py change will conflict with trunk because _oldtls.py has been deleted.

I have updated the patch based on latest trunk.

I have also fixed a new error... not sure why I did not fix it in the first
place.

```
twisted/internet/test/_win32ifaces.py:119: list comprehension redefines 'addr' from line 112
```

> Thanks again. Sorry for the long round-trip time.

No problem. Many thanks for the review.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli set owner to @adiroiban|
|-|-|

Thank you for your hard work adiroiban!

1.  The patch does not apply cleanly - apologies, but would you mind making the changes against trunk again?
1.  In test_qtreactor.py, requireModules is passed the string "twisted.interne.qtreactor" instead of "twisted.internet.qtreactor" (missing a 't' in internet)
1.  In twisted/internet/test/test_default.py, the requireModules line seem to have an extra indent
1.  In twisted/internet/_posixserialport.py, there should be 3 lines between the imports and the class definition

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli commented|
|-|-|

Oy, sorry, got my WikiFormatting wrong.    Trying again, formatted correctly this time:

 1.  The patch does not apply cleanly - apologies, but would you mind making the changes against trunk again? 
 1.  In test_qtreactor.py, requireModules is passed the string "twisted.interne.qtreactor" instead of "twisted.internet.qtreactor" (missing a 't' in internet)
 1.  In twisted/internet/test/test_default.py, the requireModules line seem to have an extra indent 
 1.  In twisted/internet/_posixserialport.py, there should be 3 lines between the imports and the class definition

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)|@adiroiban removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_7"></a>|@adiroiban commented|
|-|-|

> 1.  The patch does not apply cleanly - apologies, but would you mind
> making the changes against trunk again?

Not sure what went wrong since I did not got any conflict when merging
with latest trunk.


Uploaded a new patch based on latest trunk. Thanks!

Here is my branch on github 
https://github.com/chevah/twisted/compare/7006-pyflakes-internet

and live patch

https://github.com/chevah/twisted/compare/7006-pyflakes-internet.diff

>  1.  In test_qtreactor.py, requireModules is passed the string
> "twisted.interne.qtreactor" instead of "twisted.internet.qtreactor"
> (missing a 't' in internet)

Fixed. Thanks

>  1.  In twisted/internet/test/test_default.py, the requireModules line
> seem to have an extra indent

Fixed.

>  1.  In twisted/internet/_posixserialport.py, there should be 3 lines
> between the imports and the class definition

Done.
I have also removed the '#twisted import' comment which was stating the obvious.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun commented|
|-|-|

(In [42364]) Branching to 'internet-flakes-7006'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_10"></a>|@exarkun commented|
|-|-|

(In [42365]) Apply 7006-3.diff

refs [#7006](https://github.com/twisted/twisted/issues/7006)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_11"></a>|@exarkun set owner to @adiroiban<br>@exarkun set status to `new`|
|-|-|

Checked the patch into a branch, noticed there are some build failures.  I think some of the requireModules transformations are probably not quite correct.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_12"></a>|@adiroiban removed owner|
|-|-|

Thanks for the review. Sorry for failing tests!

I have fixed the bad usage of requireModule.

I tried to use skipUnless decorator together with requireModule, but I failed to do it so using trial and python 2.6 :)

I have updated the [#7170](https://github.com/twisted/twisted/issues/7170) ticket with feedback from this experience

I have attached a new patch based on latest trunk.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_13"></a>|@glyph commented|
|-|-|

(In [43190]) Branching to pyflakes-7006.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_14"></a>|@glyph set owner to @adiroiban|
|-|-|

Sorry again about the terrible round-trip time.  5 months :-(.

Even worse, the patch does not apply cleanly again to current trunk, so all I can say review-wise is "please make a new patch".

I think even smaller patches might make clean-ups like these easier to merge; one module at a time would be less likely to encounter conflicts over these long review cycles, at least.

Once again, thanks for persevering and sorry for the lag.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_16"></a>|@adiroiban removed owner|
|-|-|

I have attached a new full patch based on latest trunk

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_17"></a>|@exarkun set owner to @adiroiban|
|-|-|

Thanks.

  1. The changes in `twisted/internet/test/test_unix.py` are mostly wrong.  Setting the `skip` attribute to `False` is not the same as not setting it.
  1. `twisted/internet/test/test_qtreactor.py` could probably used `namedModule` instead of `import` and a seemingly pointless statement to avoid the pyflakes error.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_18"></a>|@adiroiban removed owner|
|-|-|

Thanks for the review.

I hope I have fixed these errors. I have set skip to None.
When setting skip to `False` I can see that the tests are not skipped.

I have also fixed a new one:

```
twisted/internet/test/test_endpoints.py:2590: redefinition of unused 
'test_sslDHparameters' from line 2569
```

I have attached a new diff based on latest trunk.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_19"></a>|@glyph commented|
|-|-|

(In [43779]) Branching to pyflakes-7006-2.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_20"></a>|@glyph set owner to @adiroiban|
|-|-|

Hi Adi,

Thanks for the patch.

Unfortunately it looks like the patch contains at least one typo that causes a test failure, and it conflicts a bit with trunk.

Looking forward to the next review being super quick because you can make your own branch and build it :).

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_21"></a>|@adiroiban commented|
|-|-|

(In [43851]) Branching to pyflakes-internet-7006-3.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_22"></a>|@adiroiban removed owner|
|-|-|

Hi,

Thanks for the review. My bad for not running the tests on local system.

Looks good on my computer
```
$ pyflakes twisted/web
$ ./bin/trial twisted/web
```


I have merged the old branch with trunk but I did not see any conflicts.

I sent the new branch to buildbot. Hope all is ok.

Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_24"></a>|@adiroiban commented|
|-|-|

Documentation tests are failing for an unknown reasons, as this diff does not touches does docstrings.

I think that Pyflakes is failing due to an old version of pyflakes on the builder. It pass on my local computer with latest `.[dev]` installed.

Twistedchecker are failing since twistedchecker is stupid... I tried to fix those few errors are red builders are ulgy.

Leaving this for review... and hope that pyflakes builder will be updated soon

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_25"></a>|@wsanchez set owner to @adiroiban|
|-|-|

`pydoctor` errors are a false positive for "newness" because you moved some code around.

The only "new" `pyflakes` error is:
```
twisted/internet/test/test_default.py:49: redefinition of unused 'install' from line 15
```
This exists on trunk so also appears to be a false positive.

The only `twistedchecker` error is `detectLinuxBrokenPipeBehavior` lacking an `@return`.  You changed the code to return something where it did not before, so that one seems legit, no?

If you add the `@return` and a news file, you're good to merge.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_26"></a>|@wsanchez commented|
|-|-|

Note there was an error in Debian.  I confirmed that was a transient error by forcing a rebuild and it went green.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_27"></a>|@adiroiban removed owner|
|-|-|

Many thanks for the review.

I will add the newsfile.

This patch is about cleaning all pyflakes errors from twisted/internet... so the error from test_default  should be fixed.

With this patch applied ```  pyflakes twisted/internet/ ``` should be clean.

I have updated the docstring and fixed the other pyflakes error.

Please take another look on the latest diff as I am not sure if the docstring is right.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set owner to @wsanchez<br>@wsanchez set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_29"></a>|@wsanchez set owner to @adiroiban<br>@wsanchez set status to `new`|
|-|-|

Well, you get another "new" pyflakes error same as the last one you just fixed:

```
twisted/internet/test/test_default.py:58: redefinition of unused 'install' from line 15
```

`install` is redefined in a few other places (eg. `test_unix`, `test_linux`) as well.  I'm not sure why pyflakes is only noting one at a time, even in the [full scan](https://buildbot.twistedmatrix.com/builders/pyflakes/builds/2240/steps/pyflakes/logs/pyflakes%20errors), though I supposed that's related to why these are flagged as new.

The rest, including the doc change looks good.  Feel free to fix the other redefinitions of `install` in the same manner and merge.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_30"></a>|@wsanchez commented|
|-|-|

Whoops, one more thing.  New problem raised by `twistedchecker`:

```
C0103: 49,8:PollReactorTests.test_unix: Invalid name "install_function" for type variable (should match ([a-z_][a-zA-Z0-9]*)$)
```

That should be `installFunction`.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_31"></a>|@exarkun commented|
|-|-|

1.  The change introduces a new public interface in some reactor modules, `posixInstall`.  Please don't introduce new public interfaces for no reason apart than to get rid of some pyflakes warnings.

2. In `twisted/internet/test/test_inotify.py`, don't rely on `None` evaluating to False.  If you want to check if something is None, check if it is None (you've done this correctly elsewhere).

3. In `twisted/internet/test/test_unix.py` you've made the skip definition for `test_sendFileDescriptor` **worse**.  Setting the skip attribute to None is not the same as not setting it.  Please undo this (and any other similar changes you might have made elsewhere).

4. The change in `twisted/internet/process.py` from an informative name (`ttyname`) to a garbage name (`ignore`) makes the code worse.  This warning is a pyflakes bug.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_32"></a>|@adiroiban removed owner|
|-|-|

Thanks for the review!

1. I have removed all changes from twisted/internet/gtk2reactor.py . Trunk version is accepted by the pyflakes version specified in .[dev] but I think that the buildslave runs an older version.

2. Update to explicit check for None. Sorry for that, will take some time to get this right as I am used with fuzzy false matching.

3. Done. Sorry for that!

4. Removed my changes for ttyname. But why is this a bug in pyflakes?

------


```
twisted/internet/test/test_default.py:49: redefinition of unused 'install' from line 15
```

Please also try with pyflakes 0.8.1 on your local systems.
I did not get this error on my local system.

----

I don't know what are these pydoctor errors

```
twisted.internet.unix._SendmsgMixin._isSendBufferFull: invalid ref to sendmsg.send1msg
twisted.internet.unix._UnsuportedSendmsgMixin: invalid ref to twisted.python.sendmsg
```

-------

test_runUntilFailure failed again on XP... hope [#6027](https://github.com/twisted/twisted/issues/6027) will be fixed soon.

Please take a look at latest changes. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_33"></a>|@exarkun commented|
|-|-|

This:

```
twisted.internet.unix._SendmsgMixin._isSendBufferFull: invalid ref to sendmsg.send1msg
```

I suppose is because pydoctor cannot resolve `sendmsg` to anything.  Presumably this documentation means to refer to `twisted.python.sendmsg`.  I don't know if this is a markup error by itself or if it is related to the next problem.

```
twisted.internet.unix._UnsuportedSendmsgMixin: invalid ref to twisted.python.sendmsg
```

Since `twisted.python.sendmsg` is actually a module that exists, you would think this would be a valid reference.  However, it is a module implemented in C.  I'm not entirely sure but I strongly suspect that pydoctor cannot (at least, not automatically, not the way Twisted uses it) resolve references to C modules or things defined in C modules.

The fact that https://twistedmatrix.com/documents/current/api/twisted.python.sendmsg.html is a 404 seems like it supports this conclusion.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_34"></a>|@wsanchez set owner to @adiroiban|
|-|-|

Reviewed.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_35"></a>|@adiroiban set status to `closed`|
|-|-|

(In [43942]) Merging pyflakes-internet-7006-3: Clean pyflakes for twisted internet.

Author: adiroiban
Reviewer: exarkun, cyli, glyph, wsanchez
Fixes: [#7006](https://github.com/twisted/twisted/issues/7006)
